tests/testfiles/lunamark-markdown/tex-math-dollars.test

```diff
@@ -0,0 +1,148 @@
+\def\markdownOptionTexMathDollars{true}
+\let\hat=\relax
+\let\int=\relax
```

Comment:
In the interest of future-proofing and consistency, please undefine also the remaining commands used (`\left`, `\xi`, `\right`, ...).